src/d3a/models/strategy/settlement_strategies/__init__.py

```diff
@@ -0,0 +1,142 @@
+from d3a_interface.constants_limits import ConstSettings
```

Comment:
Please don't forget the licence string (or are we not caring about this any more because we will not divide the d3a further?)

Reply:
Also, I am hesitant to request doc_strings in this module. IMO all class and method names are self-explanatory. So, feel free to skip it, but if you could bother, please add them (also in other parts of the new code). Thanks!

Reply:
Let me do both just for consistency actually.